Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/Detector.h

```diff
@@ -306,7 +308,7 @@ class Detector : public o2::Base::Detector
     GeometryHandler *mGeometryHandler;
     MisalignmentParameter *mMisalignmentParameter;
 
-    V1Layer **mGeometry;   //! Geometry
+    V3Layer **mGeometry;   //! Geometry
```

Comment:
would it be feasible to use stl containers here? Is there any strong reason to stay with `**`?